internal/common/validations.go

```diff
@@ -3,7 +3,38 @@
 
 package common
 
+import (
+	"strings"
+
+	"github.com/go-openapi/swag"
+)
+
 // IsAgentCompatible checks if the given agent image is compatible with what the service expects.
 func IsAgentCompatible(expectedImage, agentImage string) bool {
 	return agentImage == expectedImage
 }
+
+var NonIgnorableHostValidations []string = []string{"connected", "has-inventory", "machine-cidr-defined", "hostname-unique", "hostname-valid"}
+var NonIgnorableClusterValidations []string = []string{"api-vips-defined", "ingress-vips-defined", "all-hosts-are-ready-to-install", "sufficient-masters-count", "pull-secret-set", "cluster-preparation-succeeded"}
+
+var MayIgnoreValidation = func(validationID string, nonIgnorables []string) bool {
+	if validationID == "all" {
+		return true
+	}
+	return !swag.ContainsStrings(nonIgnorables, validationID)
+}
+
+var MayIgnoreValidations = func(validationIDs []string, nonIgnorables []string) (bool, string) {
```

Comment:
Why is this not a simple function? Why var?

Reply:
Good point, just fixed that.